### ISSUE TITLE
Yuzu Starts In Window Mode With Keyboard

### ISSUE THREAD

**Issue by titan-91** [ruanformigoni/gameimage#1]
After successfully building an appimage using Yuzu, it does not start in full screen mode. When agape opens Yuzu during the install process, I checked the display settings and saw where agape configured this for exclusive full screen. But when running the final appimage in Steam Deck deck mode or desktop mode, it runs as a window. I can then choose with the mouse to enable full screen in the Yuzu view menu.

Also while in the Yuzu settings I checked the controls and changed the controller from keyboard to the Steam controller. After doing this controls work fine on the Steam Deck.

**Comment by ruanformigoni** [comment]
Hi, thanks for the report! I've released a new version, `v0.2.2`, to address this issue. Could you rebuilt the appimage and see if this fixes it? You can extract the appimage with `my-game.AppImage --appimage-extract`.

**Comment by titan-91** [comment]
Sure, I will try in a couple of weeks as the Steam Deck I tested on is not mine. Thanks.

**Comment by titan-91** [comment]
Hi, so I tried the new 0.2.4 release on the Steam Deck. When Yuzu opened when running GameImage, I checked the settings. While full screen was still unchecked in the view menu, exclusive full screen was the default full screen mode which is fine. The controls were still defaulting to keyboard so I set it to the Steam option, said something like "Steam Controller 0." Then I closed Yuzu and let GameImage continue to create the appimage file. After that Yuzu now runs fine in Steam deck mode after adding the appimage file to the list of non-Steam games in the desktop version of Steam.
